packages/chart/src/scss/main.scss

```diff
@@ -133,7 +133,7 @@
   .subtext--responsive-ticks,
   .section-subtext {
     &--brush-active {
-      margin-top: 3em !important;
+      margin-top: 5rem !important;
```

Comment:
In the same place you're adding --brush-active can you use a bootstrap margin helper?

Reply:
Bootsrap does does not support large margins. I tryied mt-10 or mt-8 did not work. only mt-4 supported but that is very low amount